docs/modules/operation/pages/admin/webui/grafana-dashboard-box.adoc

```diff
@@ -72,6 +72,26 @@ When no tag is given all dashboards will be displayed
 | Maximum number of entries to be displayed (0 for unlimited)
 | Integer
 | 0
+
```

Comment:
Please update the screenshot with the attached one
![Screenshot 2022-04-29 at 11-36-02 OpenNMS Web Console](https://user-images.githubusercontent.com/1209268/165978246-f8af6177-be87-4c91-9401-b19d068c645a.png)
.